### ISSUE TITLE
Map and test button styles with the new USWDS palette

### ISSUE THREAD

**Issue by cmajel** [cloud-gov/cg-site#1540]
In our [UX audit](https://docs.google.com/document/d/1ItGl3CKfGf9BHPDA78UOLFNnrpOYl3TJC3y6c-K7JT8/edit#heading=h.cwqrt4ibffp9), we noted inconsistencies with the button styling, specifically:
* Buttons with the same background color would have different icon coloring
* Hover states not meeting WCAG AA compliance
* Inconsistent sizing
* Consider thicker font weights

We also want to test how our new palette (#1534) maps to Cloud.gov button styles and run contrast testing.

## Done when
* [x] Updated button styles are proposed
* [x] Team gives feedback
* [x] Updated button styles integrated into the site.

**Comment by cmajel** [comment]
@lauraGgit here's what I am thinking for primary and secondary button styles and how they map to the updated palette. Including a pdf export with selectable text. 

A note that in testing this I updated the palette configuration slightly. [Notes are in the color issue](https://github.com/18F/cg-site/issues/1534#issuecomment-584701331). 

![cloudgov--buttons-1](https://user-images.githubusercontent.com/2374206/74252282-70020f80-4cbb-11ea-8d8f-af6738bbe100.jpg)
[cloudgov--buttons-1.pdf](https://github.com/18F/cg-site/files/4186531/cloudgov--buttons-1.pdf)

**Comment by lauraGgit** [comment]
@cmajel per the discussion around the search - would you mind providing the widths as width design tokens rather than grid-cols?

**Comment by lauraGgit** [comment]
@cmajel also - from the looks of this there isn't a border radius on these, is that correct?

**Comment by cmajel** [comment]
@lauraGgit per slack convo, the closest value for search width was `..width-card-lg`

Re border radius the buttons have a slight 1px border radius.
